go/interpreter/lfvm/memory.go

```diff
@@ -176,47 +168,35 @@ func (m *Memory) Set(offset, size uint64, value []byte) error {
 			return errGasUintOverflow
 		}
 		if offset+size > m.Len() {
-			return fmt.Errorf("memory too small, size %d, attempted to write %d bytes at %d", m.Len(), size, offset)
+			return errSetMemTooSmall(m.Len(), size, offset)
 		}
 		copy(m.store[offset:offset+size], value)
 	}
 	return nil
 }
 
+func errSetMemTooSmall(memSize, size, offset uint64) error {
```

Comment:
renamed to `errInsufficientMemory`

Reply:
A problem of the function name is that it does not start with a verb. A name like this would be associated to a constant.

Reply:
mmmm I see. Well my idea for this is to be a sort of error generator, that can be used to compare with `errors.Is`, or `errors.As`, to be a sort of "group" of errors, and this can not be made a constant. Perhaps it could be `makeInsufficientMemoryErr` instead. wdyt ?